pkg/controller/core.oam.dev/v1alpha2/applicationconfiguration/apply_test.go

```diff
@@ -124,32 +124,32 @@ func TestApplyWorkloads(t *testing.T) {
 	}
 
 	cases := map[string]struct {
-		reason         string
-		client         resource.Applicator
-		updatingClient resource.Applicator
-		rawClient      client.Client
-		args           args
-		want           error
+		reason             string
+		patchingApplicator applicator.PatchingApplicator
+		updatingApplicator applicator.UpdatingApplicator
```

Comment:
why we still need two? I can only see `applicator.Applicator` above.

Reply:
just to change the existing codes as little as possible

Reply:
Is it easy to merge these two later?

Reply:
you mean merge like [this](https://github.com/oam-dev/kubevela/blob/7b18ef840b4889159cbef2f3a62ab51a2f1bbb38/pkg/controller/core.oam.dev/v1alpha2/applicationconfiguration/apply_test.go#L485) ?

Reply:
no, I mean merge into one applicator